lib/perlcritic/Perl/Critic/Policy/HashKeyQuotes.pm

```diff
@@ -0,0 +1,34 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package Perl::Critic::Policy::HashKeyQuotes;
+
+use strict;
+use warnings;
+use feature 'signatures';
```

Comment:
```suggestion
use experimental 'signatures';
```
Otherwise we'll get a warning that that feature is experimental (in perl versions where it still was experimental).